src/debug.rs

```diff
@@ -148,6 +209,11 @@ fn draw_text_debug(
     gizmos.line_2d(from, to, Color::RED);
 }
 
+/// A helper method to draw a rectangle
+fn draw_rect(gizmos: &mut Gizmos, origin: Vec2, size: Vec2) {
+    gizmos.rect_2d(origin + size / 2.0, 0.0, size, Color::WHITE);
```

Comment:
If this is in world space? Would it show correctly if you rotated it?

Reply:
Let's fix this in 0.2.1

Reply:
> Let's fix this in 0.2.1

Okay. Sounds good! I'll make an issue for it